tasks/e2e-old-node.sh

```diff
@@ -16,6 +16,7 @@ temp_app_path=`mktemp -d 2>/dev/null || mktemp -d -t 'temp_app_path'`
 
 function cleanup {
   echo 'Cleaning up.'
+  ps -ef | grep 'verdaccio' | grep -v grep | awk '{print $2}' | xargs kill -9
```

Comment:
This one isn't needed as verdaccio isn't used in this script.

Reply:
sorry my bad, forgot to search in this file. rolling it back.

Reply:
Done rolled-back. You can verify now. Thank for pointing it out.